administration/intro.adoc

```diff
@@ -103,42 +103,43 @@ Once Kubernetes is deployed, you will need to deploy the KubeVirt
 add-on. There are 2 ways to deploy the add-on to a cluster using the `kubectl` tool and
 manifest files:
 
-1. Stable flow
+1. Primary flow
 +++++++++++++++
 
-The stable flow installs KubeVirt using a manifest file, which contains all deployments and configurations of the
-KubeVirt core components:
+The primary flow installs KubeVirt using a operator, which manages lifecycle of all KubeVirt core components:
```

Comment:
I think it would be worth nothing that the primary flow with updates is supported starting with version 0.17.0 and greater. 

I know we had the operator in previous releases, but this is the first release we can actually support updates.

Reply:
Can you provide some example command to update from release to release and I will add it under the PR?

Reply:
this would update from whatever the current version is to the future "v0.18.0" release. 

```kubectl patch kv kubevirt -n kubevirt --type=json -p '[{ "op": "add", "path": "/spec/imageTag", "value": "v0.18.0" }]'```